tests/test_strategy.py

```
@@ -528,6 +582,175 @@ def test_classic_bandit_mixed_actions(
+
+    Parameters
+    ----------
+    a_list_str : list
+        List of action IDs.
+    a_list_float : list
+        List of prices.
+    """
+    assume(len(a_list_str) == len(a_list_float))
+    a_list_float_0_1 = [float(i) / (sum(a_list_float) + 1) for i in a_list_float]
+
+    p = dict(zip(a_list_str, a_list_float_0_1))
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)